pkg/api/dashboard_snapshot_test.go

```diff
@@ -198,6 +201,61 @@ func TestDashboardSnapshotApiEndpoint(t *testing.T) {
 					So(sc.resp.Code, ShouldEqual, 500)
 				})
 			})
+
+			Convey("Should be able to read a snapshot's un-encrypted data", func() {
```

Comment:
This is a FYI if you didn't know or anyone told you. We want to avoid introducing new GoConvey tests and use the Go standard test library, for info [here](https://github.com/grafana/grafana/blob/master/contribute/style-guides/backend.md#testing).

For these tests that uses `loggedInUserScenarioWithRole` they're very tied to GoConvey so hard to not introduce new ones and re-writing the existing is probably quite a bit of work. But good to be aware of these things moving forward.

Reply:
Will take care of the `sql` tests in this PR but I reckon will follow up in the future for this one for the reasons you mentioned